### PR TITLE
remove unnecessary `<T>` from `valarray`.

### PR DESCRIPTION
**Pull request by faithandbrave**
add `<T>` of return type for consistency.

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -6885,7 +6885,7 @@
 \indexlibrary{\idxcode{operator=}!\idxcode{valarray}}%
 \indexlibrary{\idxcode{valarray}!\idxcode{operator=}}%
 \begin{itemdecl}
-valarray& operator=(initializer_list<T> il);
+valarray<T>& operator=(initializer_list<T> il);
```
Comment (Quuxplusone) [inline source/numerics.tex]: This `<T>` should be removed (i.e., this line shouldn't be part of the diff).

Comment (faithandbrave) [inline source/numerics.tex]: Thanks for your review. I removed the `<T>`.

**Comment by jwakely** [comment]
Thanks - I committed a squashed merge of your changes.

**Comment by faithandbrave** [comment]
Thanks!

**Pull request closed without merging**